setup.py

```diff
@@ -25,13 +25,13 @@ def get_version(rel_path):
 def get_extra_deps(rel_path):
     here = os.path.abspath(os.path.dirname(__file__))
     with open(os.path.join(here, rel_path)) as fp:
-        return json.load(fp)
+        return json.load(fp)["extra_dependencies"]
 
 
 # use techniques described at https://packaging.python.org/en/latest/guides/single-sourcing-package-version/
 # Don't use technique 6 since it needs extra dependencies.
 VERSION = get_version("olive/__init__.py")
-EXTRAS = get_extra_deps("olive/extra_dependencies.json")
```

Comment:
it seems this file could be removed?

Reply:
Yes, as part of a follow up PR to avoid docs build from failing. WIP: #1051